Client/Frontend/PasswordManagement/BreachAlertsDetailView.swift

```diff
@@ -130,6 +132,8 @@ class BreachAlertsDetailView: UIView, ThemeApplicable {
             infoStack.leadingAnchor.constraint(equalTo: contentStack.leadingAnchor, constant: titleIconContainerSize),
             infoStack.trailingAnchor.constraint(equalTo: contentStack.trailingAnchor),
 
+            descriptionLabel.bottomAnchor.constraint(equalTo: goToLabel.topAnchor, constant: UX.verticalSpacing),
```

Comment:
We should avoid adding constriants in stack views. The purpose of stack vies are for helping us get rid of constraints, even tough in this class, this rule is broken multiple times. As you can see, there is aleady set a custom spacing after te description label `infoStack`: `stack.setCustomSpacing(8.0, after: self.descriptionLabel)`. The problem is that after the changes we made, the stack view does not see the views layout until we told so. The solution would be to remove `stack.setCustomSpacing(8.0, after: self.descriptionLabel)` and add the spacing after the views are set in the `infoStack`, meaning, after `[breachDateLabel, descriptionLabel, goToLabel].forEach(infoStack.addArrangedSubview)`